dingo/experimental_gurobi_functions/redundant_facet_removal_experiment.py

```diff
@@ -0,0 +1,305 @@
+# dingo : a python library for metabolic networks sampling and analysis
+# dingo is part of GeomScale project
+
+# Copyright (c) 2021 Apostolos Chalkis
+
+# Licensed under GNU LGPL.3, see LICENCE file
+
+import sys
+import numpy as np
+import scipy.sparse as sp
+import gurobipy as gp
+from gurobipy import GRB
+import math
+
+
+
+    
+
+    
+
+
+
+
+
```

Comment:
Could you plz delete all these empty lines?

Reply:
ok